eclair-core/src/main/scala/fr/acinq/eclair/payment/receive/MultiPartHandler.scala

```diff
@@ -100,41 +125,41 @@ class MultiPartHandler(nodeParams: NodeParams, db: IncomingPaymentsDb, commandBu
         }
       }
 
-    case MultiPartPaymentFSM.MultiPartHtlcFailed(paymentHash, failure, parts) if doHandle(paymentHash) =>
+    case e@MultiPartPaymentFSM.MultiPartPaymentFailed(paymentHash, failure, parts) if doHandle(paymentHash) =>
       Logs.withMdc(log)(Logs.mdc(paymentHash_opt = Some(paymentHash))) {
-        log.warning(s"payment with paidAmount=${parts.map(_.payment.amount).sum} failed ($failure)")
+        log.warning(s"payment with paidAmount=${parts.map(_.amount).sum} failed ($failure)")
```

Comment:
As a best practice we should use less string interpolation for logs and more `"{}", myvar`

Reply:
I think it's especially useful for `log.debug` and `log.info` that may be turned off, for `warn` and `error` which you should always keep this isn't an issue. I'll update this file though.